docs/Administration.rst

```diff
@@ -121,15 +119,26 @@ browser supports the HTML 5 and the Drag and Drop function.
 
 .. note::  Even if you delete your company logo, the default logo will be displayed..
 
-3. Click the box in the Select Navigation Bar Style pane and select one
-   style from the two choices displayed in radio buttons.
+3. Click on Primary, Secondary and Tertiary color pickers so that your style selection from a color picker will also make the color 
+   of buttons and backgournd.
+
+|image364|
+
+You can use the color picker right to select a color and get the HTML Color Code of this pixel. Also you get the HEX color code value of the selected color or put on your HEX and it will be automatically identified, or even you can select a predefined color from the palette.
```

Comment:
```suggestion
You can use the color picker right to select a color and get the HTML color code of this pixel. Also you get the hexadecimal color code value of the selected color or directly enter the hexadecimal code and it will be automatically recognized. Finally, you can also select a predefined color from the palette.
```